UPDATING.md

```diff
@@ -24,6 +24,17 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Renamed "extra" requirments for cloud providers
+
+Subpackages for specific services have been combined into one variant for
+the cloud provider.
+If you want to install integration for Microsoft Azure, then instead of
+`pip install airflow[azure_blob_storage,azure_data_lake,azure_cosmos,azure_container_instances]`,
+you should execute `pip install airflow[azure]`.
+If you want to install integration for Amazon Web Services, then instead of
+`pip install airflow[s3,emr]`, you should execute `pip install airflow[aws]`.
+The integration with GCP is unchanged.
```

Comment:
The `_api` suffix is out of place, so please can we rename `gcp_api` to simply `gcp`?

Reply:
I want to do it in a separate PR, because I do not think that everyone will agrees with this change. This PR is responsible only for reducing the number of packages in the case of azure and aws. In the next PR I will introduce your change and I personally support it, but I do not want to introduce any unrelated changes in this PR. If this change is accepted, it will immediately prepare another one, which will introduce the proposed change and will also update `UPDATING.md`.

Look at previous discusion: https://github.com/apache/airflow/pull/4524/files#r249197669